source/darwin/securityframework_ecc.c

```diff
@@ -21,6 +21,29 @@ static uint8_t s_preamble = 0x04;
 
 static size_t s_der_overhead = 8;
 
+/* The hard-coded "valid" public keys. */
```

Comment:
nit: add a note that its the same public keys we use for tests. dont want anyone getting confused about us checking in real keys into source code